src/renderer/packages/remote.ts

```diff
@@ -167,41 +181,49 @@ export interface DialogConfig {
 }
 
 export async function getDialogConfig(params: { uuid: string; language: string; version: string }) {
-  type Response = {
+  type DialogConfigResponse = {
     data: null | DialogConfig
   }
-  const res = await ofetch<Response>(`${API_ORIGIN}/api/dialog_config`, {
+  const res = await afetch(`${API_ORIGIN}/api/dialog_config`, {
     method: 'POST',
+    body: JSON.stringify(params),
+  }, {
     retry: 3,
-    body: params,
   })
-  return res['data'] || null
+  const json: DialogConfigResponse = await res.json()
+  return json['data'] || null
 }
 
 export async function getLicenseDetail(params: { licenseKey: string }) {
-  type Response = {
+  type LicenseDetailResponse = {
     data: ChatboxAILicenseDetail | null
   }
-  const res = await ofetch<Response>(`${API_ORIGIN}/api/license/detail`, {
-    retry: 3,
+  const res = await afetch(`${API_ORIGIN}/api/license/detail`, {
```

Comment:
I'm not sure what are the internal differences between `afetch` and `oftech`, need to check carefully.